RELEASE/scripts/sl_ascend/sl_mr2018.ash

```diff
@@ -702,7 +702,14 @@ boolean neverendingPartyAvailable()
 	{
 		return false;
 	}
+	if(get_property("sl_skipNEPOverride").to_boolean())
```

Comment:
Thanks for adding this! I think the behavior we want here is closer to "Don't use the NEP on the last day of the ascension, but we're OK with using it on earlier days." In that case, changing this condition to `get_property("sl_skipNEPOverride").to_boolean() && my_level() <= 10` would (probably) not use the NEP on the last day of the ascension.

The other alternative is to make it a daily-resetting variable, `_sl_skipNEPToday`, that people could set on the last day. But this probably has worse usability - just bringing it up as an option in case it fixes your particular problem.

Reply:
I worry about linking it to the level also. Since we don't know what shinies may come out in the future, what pathways may come out in the future, or any other variables (such as someone skipping a day and having extra ADV on the final day), I think overriding something that is explicitly an override is just begging for bug reports.

Reply:
Yup, sounds reasonable to me - was throwing out ideas but what I think what you've got here is fine.

Reply:
Awesome Thanks. :)